src/components/places/map-with-all-places/map-with-all-places.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react'
+const { withScriptjs, withGoogleMap, GoogleMap } = require("react-google-maps")
+const { MarkerWithLabel } = require("react-google-maps/lib/components/addons/MarkerWithLabel")
+
+const MapWithAllPlaces = ({ places }) => {
+    return (
+    <GoogleMap
+      defaultZoom={13}
+      defaultCenter={{ lat: 50.0910966, lng: 14.401616500000046 }}
+    >
+    {places.map(place =>
+      <MarkerWithLabel
+        position={{ lat: place.lat, lng: place.long }}
```

Comment:
React się nie pluje, że brakuje `key`?

Reply:
nie pluje sie, bo nie mam eslinta hahaha. Jak go dodam, to bede poprawiac ;p

Reply:
ale to nie eslint tylko React w konsoli w przeglądarce. Nie ma nic?

Reply:
nic nie bylo 🤷‍♂